<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupSubscriptionDialogClass</class>
 <widget class="QDialog" name="GroupSubscriptionDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Group Subscriptions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../siilihairesources.qrc">
    <normaloff>:/data/folder-new.png</normaloff>:/data/folder-new.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Please choose the groups you wish to subscribe to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="selectAllButton">
         <property name="text">
          <string>Select all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectNoneButton">
         <property name="text">
          <string>Select none</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../siilihairesources.qrc"/>
  <include location="../../siilihairesources.qrc"/>
  <include location="../../siilihairesources.qrc"/>
  <include location="../../siilihairesources.qrc"/>
 </resources>
 <connections/>
</ui>
